Makefile

```diff
@@ -51,11 +51,10 @@ endef
 define make_docker
 	$(eval svc=$(subst docker_,,$(1)))
 
-	docker build \
+	docker buildx build --load \
+		--platform linux/amd64 \
```

Comment:
Let's add a Makefile target for building and tagging dockers for different architecture.